<?xml version="1.0"?>
<launch>

    <arg name="num_of_object" default="25"/>
    <arg name="sensor_topic" default="/cloud_without_segmented"/>
    <arg name="mesh_base_topic" default="meshcloud"/>
    <arg name="output_topic_base" default="all_cloud"/>
    <arg name="radius" default="10000"/>
    <arg name="num_of_nearest_points" default="7"/>
    <arg name="photoneo_sensor_img" default="/photoneo_center/sensor/image_color"/>
    <arg name="img_service_name" default="input_img"/>
    <arg name="timespan" default="4"/>
    <arg name="main_service_name" default="third_input"/>
    <arg name="photoneo_frame_id" default="photoneo_center_optical_frame"/>
    
    <node name="nearest_search_bara" pkg="ishiyama_annotation" type="nearest_search_ishiyama_img"  output="screen" respawn="false" respawn_delay="0">
        <param name="num_of_object" value="$(arg num_of_object)"/>
        <param name="sensor_topic" value="$(arg sensor_topic)"/>
        <param name="mesh_base_topic" value="$(arg mesh_base_topic)"/>
        <param name="output_topic_base" value="$(arg output_topic_base)"/>
        <param name="radius" value="$(arg radius)"/>
        <param name="num_of_nearest_points" value="$(arg num_of_nearest_points)"/>
        <param name="photoneo_sensor_img" value="$(arg photoneo_sensor_img)"/>
        <param name="img_service_name" value="$(arg img_service_name)"/>
        <param name="timespan" value="$(arg timespan)"/>
        <param name="main_service_name" value="$(arg main_service_name)"/>
        <param name="photoneo_frame_id" value="$(arg photoneo_frame_id)"/>
        
    </node>

</launch>